src/modules/flow/http-client/http-client.c

```diff
@@ -457,8 +527,13 @@ common_get_process(struct sol_flow_node *node, void *data, uint16_t port,
         SOL_INT_CHECK_GOTO(r, < 0, err);
     }
 
-    connection = sol_http_client_request(SOL_HTTP_METHOD_GET, mdata->url,
-        &params, type->http_response ? : common_request_finished, node);
+    if (!is_sse_request) {
+        connection = sol_http_client_request(SOL_HTTP_METHOD_GET, mdata->url,
+            &params, type->http_response ? : common_request_finished, node);
+    } else {
+        connection = sol_http_client_request_with_interface(SOL_HTTP_METHOD_GET,
```

Comment:
you should always do this one. If server replied with `Content-Type: text/steam` then you keep the progressive approach. Otherwise you wait all the content to arrive and process as usual.